src/BundlerMinifier.TagHelpers/FileVersionProvider.cs

```diff
@@ -112,7 +111,7 @@ public string AddFileVersionToPath(string path)
 
         private static string GetHashForFile(IFileInfo fileInfo)
         {
-            using (var sha256 = CryptographyAlgorithms.CreateSHA256())
+            using (var sha256 = new System.Security.Cryptography.SHA256Managed())
```

Comment:
`HashAlgorithm.Create("SHA256")` would be better, as then the framework will pick the best implementation supported by the current OS, rather than forcing it to use the non-FIPS compliant managed implementation.

Reply:
@martincostello I was using HashAlgorithm.Create("SHA256") in my own library long ago. It turns out that it was way slower, because it is fetching some stuff in the registry or configuration to know what instance it should pick. This seems to me way overkill.

Reply:
But that stuff is there for a reason so that the OS-specific cryptographic APIs are used where appropriate and the code configuration is its most secure and can evolve over time without necessarily requiring code changes.